registry/quilt_server/views.py

```diff
@@ -1826,3 +1826,31 @@ def audit_user(user):
             extra=event.extra,
         ) for event in events]
     )
+
+@app.route('/api/admin/package_summary')
+@api(require_admin=True)
+@as_json
+def package_summary():
+    events = (
+        db.session.query(Event.package_owner, Event.package_name, Event.type, 
+                sa.func.count(Event.type), sa.func.max(Event.created))
```

Comment:
Make it aligned with the open parenthesis... or just move the args above to the next line